solutions/security/cloud/asset-disc-aws.md

```diff
@@ -0,0 +1,319 @@
+---
+applies_to:
+  stack: preview
+  serverless:
+    security: preview
+---
+
+# Set up Cloud Asset Discovery for AWS
+
+## Overview [cad-aws-overview]
+
+This page explains how to set up the Cloud Asset Discovery integration (CAD) to inventory assets in AWS.
```

Comment:
_personal taste comment_

This is the first time I hear about the acronym CAD. @caitlinbetz is it something we've been referring to already? If not, I would rather not introduce this term in documentation